pyquil/job_results.py

```diff
@@ -100,7 +100,7 @@ def recover_complexes(coef_string, classical_addresses):
     """
     num_octets = len(coef_string)
     num_addresses = len(classical_addresses)
-    num_memory_octets = _round_to_next_multiple(num_addresses, 8) / 8
+    num_memory_octets = _round_to_next_multiple(num_addresses, 8) // 8
```

Comment:
Mypy looks like a nice package!  I see that it is still below the 1.0 version.  Have you had much experience working with it in production?

On the Python3 vs. 2 note.  We're definitely moving towards switching, but haven't firmed up the timeline yet.